### ISSUE TITLE
Input type doesn't change after dynamically re-initialization 

### ISSUE THREAD

**Issue by nirob110** [wenzhixin/multiple-select#367]
// initialization at first. which works fine.
var chapter = jQuery('#chapterId').multipleSelect({
            filter: true,
            single : true
      });
// but when i re-initialize select from radio change event and set single to false, give it a refresh the 
// checkbox doesn't appear accordingly as it should, I think.
$('input[type=radio][name=examType]').change(function(){
	jQuery('#chapterId').html('');
	var chapter = jQuery('#chapterId').multipleSelect({
	        filter: true,
                single : false;
        });
        chapter.html('').append(optns).multipleSelect("refresh"); // refreshing from radio change event
});
Is it a bug or anyway not meant to be ?

**Comment by martingraham** [comment]
I found I needed to delete the old ms-form
`$("#chapterID").siblings(".ms-parent").remove();`

And then clear associated JQuery data attached to the select element
`$("#chapterID").removeData();`

Then I could reinstate a new multiple selection instance of a different selection type
e.g. your
```
var chapter = jQuery('#chapterId').multipleSelect({
filter: true,
single : false;
});
```

The selections seem to carry over between the two, which can be handy, or problematic (when you end up with a single selection dropdown with multiple things selected). You'll have to do something with setSelects to fix that

**Comment by wenzhixin** [comment]
Added `refreshOptions` method.
Fixed in #458
